verification/certifyKey.go

```diff
@@ -449,7 +451,7 @@ func testCertifyKey(d TestDPEInstance, client DPEClient, t *testing.T, simulatio
 	digestLen := profile.GetDigestSize()
 
 	seqLabel := make([]byte, digestLen)
-	for i, _ := range seqLabel {
+	for i := range seqLabel {
```

Comment:
IDE was giving a warning for empty value "simplify range expression", so removed '_'